rados/rados_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/ceph/go-ceph/rados"
+	"github.com/immesys/go-ceph/rados"
```

Comment:
modules would make this much easier

Reply:
this is annoying - changing it back before merging